python/lsst/pipe/analysis/coaddAnalysis.py

```diff
@@ -516,6 +517,13 @@ def runDataRef(self, patchRefList, subdir="", cosmos=None):
         self.allStats, self.allStatsHigh = savePlots(plotList, "plotCoadd", repoInfo.dataId,
                                                      repoInfo.butler, subdir=subdir)
 
+        metaDict = {kk: plotInfoDict[kk] for kk in ("cameraName","filter","tract","visit","patch","rerun")}
```

Comment:
You mention that `visitAnalysis` can call this and that's why you left in the irrelevant `visit` metadata, but that's not the case. `visitAnalysis` does it's own `metaDict` setting here:
https://github.com/lsst-dm/pipe_analysis/blob/master/python/lsst/pipe/analysis/visitAnalysis.py#L527-L530
(and I don't know that you can guarantee that `visit` will resolve to `None` whenever you think it should...) 
 And do note the `camera` vs. `cameraName` entry.  If you INSIST on naming the tag `cameraName`, that will be anti-pattern to what is already in SQuaSH and will leave things inconsistent within `pipe_analysis`.  I would not want to live in this state while awaiting another ticket...I'd prefer keeping things consistent and, if you feel we *must* make the switch to `cameraName`, then that should happen everywhere on this ticket or atomically on a separate ticket.

Also, I don't think the `patchList` is really appropriate as a tag (how would a list of ~81 patches look in the dropdown menu...and is a `list` there even allowed?)  I think this might be more appropriate in the `measExtrasDictList`.

Reply:
I wouldn't insist on naming it `cameraName`, but would have preferred it. I've changed it to call `camera` now. I also exclude any `patch` information, since it probably is run on all 81 patches by default.